api/envoy/config/core/v3/health_check.proto

```diff
@@ -145,6 +145,10 @@ message HealthCheck {
     // <envoy_v3_api_msg_type.matcher.v3.StringMatcher>`. See the :ref:`architecture overview
     // <arch_overview_health_checking_identity>` for more information.
     type.matcher.v3.StringMatcher service_name_matcher = 11;
+
+    // An optional string to match anywhere in the first 1024 bytes of the response body.
+    // If it is set, both the expected response check and status code determine the health check.
+    bytes response = 13 [(validate.rules).bytes = {max_len: 1024}];
```

Comment:
Please move next to `expected_statuses`.

Also renaming to `response_prefix_contents` or `response_prefix_substring` may represent this field better.

Reply:
Thank you! Done.